src/components/Discover/useTrackDiscoverScreenTime.ts

```diff
@@ -6,11 +6,12 @@ import { PerformanceMetrics } from '@/performance/tracking/types/PerformanceMetr
 
 export const useTrackDiscoverScreenTime = () => {
   const isOnDiscoverScreen = useNavigationStore(state => state.isRouteActive(Routes.DISCOVER_SCREEN));
+  const activeRoute = useNavigationStore(state => state.activeRoute);
```

Comment:
Would avoid subscribing to this since it's not being responded to, and will cause needless updates in this hook. Would replace with a `getState` call:

```ts
if (!isOnDiscoverScreen && data?.startTimestamp && useNavigationStore.getState().activeRoute !== Routes.NETWORK_SELECTOR) {
```